torch/testing/_internal/opinfo/definitions/signal.py

```diff
@@ -824,4 +824,57 @@ def make_signal_windows_opinfo(
             ),
         ),
     ),
+    make_signal_windows_opinfo(
+        name="signal.windows.nuttall",
+        ref=reference_signal_window(scipy.signal.windows.nuttall)
+        if TEST_SCIPY
+        else None,
+        sample_inputs_func=sample_inputs_window,
+        reference_inputs_func=reference_inputs_window,
+        error_inputs_func=error_inputs_window,
+        skips=(
+            DecorateInfo(
+                unittest.expectedFailure,
+                "TestDecomp",
+                "test_comprehensive",
+                dtypes=[torch.float16],
+                device_type="cpu",
+            ),
```

Comment:
I think we should disallow now that we can `float16` and `bfloat16` dtypes as they have all these weird edge cases. We can always enable them later again if people ask for them. What do you think?